### PR TITLE
Info pages #13xan #13xak

### PR DESCRIPTION
**Pull request by gdude2002**

### CODE REVIEW

**Review by lemonsaurus — DISMISSED** [review]
I got nothing. This looks great.

File: pysite/views/error_handlers/http_5xx.py [hunk 1]
```diff
@@ -13,28 +13,17 @@ class Error500View(ErrorView):
...
-                                             "again later. "
-                                             "If you believe we have made a mistake, "
-                                             "please "
-                                             "<a href='https://github.com"
-                                             "/discord-python/site/issues'>file an issue on our GitHub"
-                                             "</a>."), error.code
+        return self.render("errors/error.html", code=error.code, req=request, error_title=error_desc,
+                           error_message="An error occurred while processing this request, please try "
+                                         "again later. If you believe we have made a mistake, please "
+                                         "<a href='https://github.com/discord-python/site/issues'>file an issue on our"
```
Comment (jerbob) [inline pysite/views/error_handlers/http_5xx.py]: These lines get *very* close to 120 characters. Maybe shove everything after the `>` to the line below?

File: pysite/views/error_handlers/http_5xx.py [hunk 2]
```diff
@@ -13,28 +13,17 @@ class Error500View(ErrorView):
...
```
Comment (jerbob) [inline pysite/views/error_handlers/http_5xx.py]: These lines get *very* close to 120 characters. Maybe shove everything after the `>` to the line below?

File: static/resources.json [hunk 3]
```diff
@@ -0,0 +1,45 @@
+{
+  "Learning Resources": {
+    "description": "Tutorials and references for those that are just getting started with python",
+    "resources": {
+      "Automate the Boring Stuff with Python": {
+        "description": "One of the best books out there for Python beginners. You can buy a copy, but there's also a free online version.",
```
Comment (jerbob) [inline static/resources.json:6]: This line is 139 characters. Change pls

File: static/resources.json [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  },
+  "IDEs": {
+    "description": "Fully-integrated development environments for serious Python work",
+    "resources": {
+      "PyCharm (Community and Professional editions available)": {
+        "description": "The very best Python IDE, with a wealth of advanced features and convenience functions.",
+        "url": "https://www.jetbrains.com/pycharm/"
+      },
+      "Spyder (Free)": {
+        "description": "The Scientific PYthon Development EnviRonment. Simpler and lighter than PyCharm, but still packs a punch.",
```
Comment (jerbob) [inline static/resources.json:40]: This line is 131 characters. Change pls

**Review by jerbob — CHANGES_REQUESTED** [review]

File: app_test.py [hunk 5]
```diff
@@ -33,11 +33,32 @@ def test_index(self):
...
+        response = self.client.get('/info')
+        self.assertEqual(response.status_code, 301)
+
     def test_info_help(self):
         """ Check the info help path responds with 200 OK """
         response = self.client.get('/info/help')
         self.assertEqual(response.status_code, 200)
 
+    def test_info_resources(self):
+        """ Check the info resources path responds with 200 OK """
```
Comment (jerbob) [inline app_test.py:47]: These docstrings should have no spaces surrounding them, and end with a period.

**Review by jerbob — DISMISSED** [review]

File: static/resources.json (hunk 3 above)
Comment (gdude2002) [reply static/resources.json:6]: This is a JSON file.

File: static/resources.json (hunk 4 above)
Comment (gdude2002) [reply static/resources.json:40]: This is a JSON file.

File: app_test.py (hunk 5 above)
Comment (gdude2002) [reply app_test.py:47]: The entire test file is like that, isn't it?

**Review by jerbob — APPROVED** [review]

**Pull request merged**